docs/source/mods/sharpe-ratio.rst

```diff
@@ -0,0 +1,227 @@
+Maximum Sharpe Ratio
+====================
+
+The Sharpe ratio, named after Nobel laureate William F. Sharpe, measures the risk-adjusted performance of a portfolio. It is calculated by subtracting the risk-free rate of return from a portfolio's expected return and dividing the result by the portfolio's standard deviation. Thus, a higher Sharpe ratio corresponds to more attractive risk-adjusted performance, as the portfolio's returns are higher relative to the portfolio's risk.
+
+This OptiMod computes the portfolio that maximizes the Sharpe ratio for the given expected returns and variances.
+
+
+Problem Specification
+---------------------
+
+.. tabs::
```

Comment:
Can you tweak this section to use the "dropdown" directive for the mathsy parts (instead of tabs)? Check the bipartite matching mod for an example.